app/services/notifications.js

```diff
@@ -0,0 +1,89 @@
+import {getOwner} from '@ember/application';
+import Service, {inject as service} from '@ember/service';
+
+import NotificationsModel from '../models/notifications/notifications';
+
+export default Service.extend({
```

Comment:
This service is merged with the main app tree. How about adding a `wds-` prefix to avoid conflicts?